### PR TITLE
cloud server docker images print time for each line log

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: port/oc_log.h [hunk 1]
```diff
@@ -42,6 +42,8 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <sys/time.h>
```
Comment (Danielius1922) [inline port/oc_log.h:45]: I think `<sys/*>` headers are Unix only.

Maybe `port/oc_clock.h` or `oc_clock_util.h` might have some portable functions.

Comment (jkralik) [reply port/oc_log.h]: removed

File: CMakeLists.txt [hunk 2]
```diff
@@ -797,6 +797,9 @@ if(BUILD_TESTING AND(UNIX OR MINGW))
     endmacro()
 
     file(GLOB COMMONTEST_SRC tests/gtest/*.cpp)
+    if (OC_DEBUG_ENABLED)
+        list(APPEND COMMONTEST_SRC ${PROJECT_SOURCE_DIR}/api/oc_clock.c)
```
Comment (Danielius1922) [inline CMakeLists.txt:801]: This is not needed, no?

Comment (jkralik) [reply CMakeLists.txt]: no it is not needed - removed

**Review by Danielius1922 — APPROVED** [review]

**Comment by jkralik** [comment]
@SiMet pls could you check `ocf conformance testing`? It seems to be hanged.

**Pull request merged**